doc/man7/EVP_SIGNATURE-ED25519.pod

```diff
@@ -56,7 +56,7 @@ One of the five strings "Ed25519", "Ed25519ctx", "Ed25519ph", "Ed448", "Ed448ph"
 
 A string of octets with length at most 255.
 
-=item * "verify-digested" (B<OSSL_SIGNATURE_PARAM_EDDSA_VERIFY_DIGESTED> <integer>
+=item * "verify-digested" (B<OSSL_SIGNATURE_PARAM_FIPS_EDDSA_VERIFY_DIGESTED> <integer>
```

Comment:
@paulidale - should this be no-verify-digested ??

Reply:
Yes

Reply:
added no